src/components/TodoComponents/Todo.js

```diff
@@ -1,12 +1,35 @@
 import React from "react";
 import "./Todo.css";
 import Form from "./TodoForm.js";
+import search from "../../assets/search.png"
 class Todo extends React.Component {
+  state={
+    text:"",
+    searchText:""
+  }
+  searchHandler = () =>{
```

Comment:
For some reason, when I'm using the search functionality, I have to click the submit button twice to get the item I want. Did you consider filtering through the the list you have  and then reassigning that list to the previous list to render?